website/identifiers/tasks.py

```diff
@@ -27,7 +27,14 @@ def task__update_verified_links(self, target_guid):
                  f'[guid={target_guid}]')
 
     Guid = apps.get_model('osf.Guid')
-    target_object = Guid.load(target_guid).referent
+    guid = Guid.load(target_guid)
+
+    if guid is not None:
+        target_object = guid.referent
+    else:
+        logger.error(f"Failed to load guid= {target_guid}")
+        return
```

Comment:
In addition, should we trigger the retry instead of just return?

GV should send a valid guid, it could be the DB fluke which `load` a valid `guid` returns `None`. This happened from time to time unfortunately.